tetragon.yaml

```
@@ -48,9 +48,9 @@ pipeline:
 
   - name: Install BPF objects
     runs: |
-      mkdir -p "${{targets.destdir}}/var/lib/tetragon/bpf/"
+      mkdir -p "${{targets.destdir}}/var/lib/tetragon/"
       # Tetragon loads these at runtime from --bpf-lib (defaults under /var/lib/tetragon)
-      install -m 0644 bpf/objs/* "${{targets.destdir}}/var/lib/tetragon/bpf/"
+      install -m 0644 bpf/objs/* "${{targets.destdir}}/var/lib/tetragon/"
```

```suggestion
      install -Dm 0644 bpf/objs/* "${{targets.destdir}}/var/lib/tetragon/"
```
nit considering consistency: with install we can use `-D` to create the directory and remove explicit `mkdir` above.

---

`install -Dm 0644 bpf/objs/* "${{targets.destdir}}/var/lib/tetragon/"` leads to:
```
2025/08/14 10:17:56 WARN install: can't create '/home/build/melange-out/tetragon/var/lib/tetragon/': Is a directory
2025/08/14 10:17:56 WARN install: can't create '/home/build/melange-out/tetragon/var/lib/tetragon/': Is a directory
```

If we try with something like: `install -Dm 0644 bpf/objs/* "${{targets.destdir}}/var/lib/tetragon/*"`
```
# apk manifest --allow-untrusted packages/aarch64/tetragon-fips-1.5.0-r1.apk 
sha1:63377c7b7c4f80dd525b5c84f509abfeda2aa417  etc/tetragon/tetragon.yaml
sha1:a5c7127699ccf18f78102117d887e831eb779202  usr/bin/tetragon
sha1:e5b4b4684f2abb1494b54a9929653e61d3910bd1  var/lib/db/sbom/tetragon-fips-1.5.0-r1.spdx.json
sha1:7bc3154e460d34c4c56afa990e2ec8659f7b7418  var/lib/tetragon/* :weird:
```